sdk/identity/identity/review/identity.api.md

```diff
@@ -51,11 +51,17 @@ export class AuthenticationRequiredError extends Error {
 
 // @public
 export class AuthorizationCodeCredential implements TokenCredential {
-    constructor(tenantId: string | "common", clientId: string, clientSecret: string, authorizationCode: string, redirectUri: string, options?: TokenCredentialOptions);
-    constructor(tenantId: string | "common", clientId: string, authorizationCode: string, redirectUri: string, options?: TokenCredentialOptions);
+    constructor(tenantId: string | "common", clientId: string, clientSecret: string, authorizationCode: string, redirectUri: string, options?: AuthorizationCodeCredentialOptions);
+    constructor(tenantId: string | "common", clientId: string, authorizationCode: string, redirectUri: string, options?: AuthorizationCodeCredentialOptions);
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
     }
 
+// @public
+export interface AuthorizationCodeCredentialOptions extends TokenCredentialOptions {
+    allowMultiTenantAuthentication?: boolean;
```

Comment:
Is there a reason not to add the property directly to TokenCredentialOptions?

https://github.com/Azure/azure-sdk-for-js/blob/e823b26f4600a698eb1c06f48da9b780fb2331a7/sdk/identity/identity/src/client/identityClient.ts#L312

Reply:
The reason is whether all credentials will support it! Are we sure all credentials will?

Reply:
This property is now in all the credentials :)